### ISSUE TITLE
Search docs not showing up

### ISSUE THREAD

**Issue by adborden** [caciviclab/disclosure-backend#149]
Unable to read api 'search' from path http://127.0.0.1:8000/docs/api-docs/search (server returned INTERNAL SERVER ERROR)

Stack trace:

```
Internal Server Error: /docs/api-docs/search
Traceback (most recent call last):
  File "/data/backend/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/data/backend/env/local/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/data/backend/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework_swagger/views.py", line 151, in get
    'apis': generator.generate(apis),
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework_swagger/docgenerator.py", line 41, in generate
    'operations': self.get_operations(api, apis),
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework_swagger/docgenerator.py", line 84, in get_operations
    'nickname': method_introspector.get_nickname(),
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework_swagger/introspectors.py", line 260, in get_nickname
    return rest_framework.settings.api_settings \
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework/settings.py", line 208, in __getattr__
    val = perform_import(val, attr)
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework/settings.py", line 158, in perform_import
    return import_from_string(val, setting_name)
  File "/data/backend/env/local/lib/python2.7/site-packages/rest_framework/settings.py", line 176, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'disclosure.swagger_nickname.view_name' for API setting 'VIEW_NAME_FUNCTION'. ImportError: No module named swagger_nickname.
[pid: 20741|app: 0|req: 17/17] 198.144.220.83 () {44 vars in 860 bytes} [Tue Jan 26 02:44:54 2016] GET /docs/api-docs/search => generated 27 bytes in 11 msecs (HTTP/1.1 500) 3 headers in 106 bytes (1 switches on core 0)
```

Looks like we deleted a file.

**Comment by bcipolli** [comment]
Very interesting. I definitely deleted that file, and the tests definitely passed.

I'll check into undeleting the file and figure out why our tests didn't catch that. Assigning to: me!

**Comment by adborden** [comment]
If we want to write a test for this, I would look into one of the python
swagger spec validators at swagger.io and point it at the spec endpoint.
We can do some additional validations like every operation has a
"nickname" field.

In that case, this test would have probably failed with a 500.

On Mon, 2016-01-25 at 22:49 -0800, Ben Cipollini notifications@github.com wrote:

> Very interesting. I definitely deleted that file, and the tests definitely passed.
> 
> I'll check into undeleting the file and figure out why our tests didn't catch that. Assigning to: me!
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/caciviclab/disclosure-backend/issues/149#issuecomment-174860636

## 

Aaron Borden
Human and Hacker

**Comment by bcipolli** [comment]
Fixed in #177
